src/main/resources/db/changelog/changes/init_procedure.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+
+  <changeSet id="init_procedure" author="Firebird">
+    <createProcedure path="changes/init_procedure.sql" relativeToChangelogFile="true" />
```

Comment:
I would propose to rename script from init_procedure.sql to save_instances_ids.sql.